### ISSUE TITLE
Keywords highlighting not working

### ISSUE THREAD

**Issue by arn-all** [ThFriedrich/lammps_vscode#7]
For some reason, keywords highlighting stopped working, whilst comments and numbers are still correctly highlighted. I'll do more tests to try to find a workaround.

![](https://user-images.githubusercontent.com/45487966/79124745-49388600-7d9d-11ea-9e01-1b922b1fe297.png)

Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T11:22:13.689Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.5.15-200.fc31.x86_64

**Comment by ThFriedrich** [comment]
Can you check what you get when you inspect the scope with the command `Developer: Inspect Editor Tokens and Scopes` as suggested [here](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector)? 
![image](https://user-images.githubusercontent.com/47680554/79347553-1b397a00-7f34-11ea-9d90-b04e8378ce21.png)

**Comment by arn-all** [comment]
Using the tool helped me spot my problem. It's nothing about the extension. I was simply using the ["atom material theme"](https://marketplace.visualstudio.com/items?itemName=tobiasalthoff.atom-material-theme) for some time (can't remember when I switched), which for some reason shows the keywords white. I picked another one, now everything works fine.

I don't think we should worry too much about that, all other themes I had installed behave correctly.
